### PR TITLE
Stop stripping release names from container names

### PR DESCRIPTION
**Pull request by schnie**
We no longer prepend the release name, so we can drop the logic to strip it out now.

### CODE REVIEW

File: src/instruments/graphs/ContainerList.js [hunk 1]
```diff
@@ -35,40 +35,34 @@ class ContainerList extends React.Component {
...
             {data.map((d, i) => (
               <div
                 key={`${d.metric.container}${i}`}
                 className={s.containerListItem}>
-                <div>{this.stripName(d.metric.container)}</div>
                 <div style={{ flex: '0 0 45%' }}>
                   {d.metric.pod != undefined &&
                     `${d.metric.pod.split(/-/g)[3]}-${
                       d.metric.pod.split(/-/g)[4]
                     }-${d.metric.pod.split(/-/g)[5]}`}{' '}
```
Comment (schnie) [inline src/instruments/graphs/ContainerList.js]: @samblackk `StatefulSet` pods like the redis one are named a little differently (like `planetary-nebula-1234-redis-0`), as opposed to a normal `Deployment`, where the pods have the random id with another dash in it. I guess we can just show the full pod names and be done with it (removing this regex logic), or update the logic to deal with the difference. What do you think?

Comment (schnie) [reply src/instruments/graphs/ContainerList.js]: @ryw any thoughts? Show full pod names or keep trimming them?

Comment (schnie) [reply src/instruments/graphs/ContainerList.js]: Here's what it looks like if we drop the regex formatting here.

Comment (schnie) [reply src/instruments/graphs/ContainerList.js]: (also shows the new container name format)

Comment (samblackk) [reply src/instruments/graphs/ContainerList.js]: @schnie I prefer the raw name over the trimmed version. Gives the user complete information.

**Pull request merged**